src/main.js

```diff
@@ -0,0 +1,10 @@
+import App from './App.svelte';
+
+const app = new App({
+	target: document.body,
+	props: {
+		name: 'world'
```

Comment:
This prop is not used. You can delete it.

Reply:
deleted